core/dbt/task/deps.py

```diff
@@ -55,39 +96,149 @@ def track_package_install(
         )
 
     def run(self) -> None:
+        if system.path_exists(self.project.packages_install_path):
```

Comment:
Do you think this could cause any surprise behavior that's different from how deps works today?